src/features/patients/hooks/useGetMdiPatientApp.hook.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { ReactQueryKeys } from "constants/keys";
+import useCurrentPatient from "fetchDataHooks/useCurrentPatient";
+import api from "capableApi/index";
+
+export const getMdiPatientApp = async (patientId: string) => {
+  const response = await api.client.Patient.mdiPatientApp(patientId);
+
+  if (!response.ok) {
+    const error = `Error retrieving resource ${response.error}`;
+    console.error(error);
+    throw new Error(error);
+  }
+
+  return response;
```

Comment:
```suggestion
  return response as MdiPatientApp;
```

would be nice to add the type here